### PR TITLE
Added toast on delete action

### LINKED ISSUE #1156: Adding "Advertisement Deleted" toast on delete action
**Describe the bug**
We follow toast-notification on create , update and delete actions that toast is not implemented on delete ads action.

**To Reproduce**

1. click on manage button on any organization from the org list 
2. click on advertisement from the menu
3.  click on delete button.

**Expected behavior**
Delete toast notification 

**Actual behavior**
NA

**Screenshots**
Screenshot 2023-12-07 at 7 59 07 PM

**Additional details**
Ps:  please also change the color of delete button as per the standard format in the application.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Min2who**
**What kind of change does this PR introduce?**

Bug

**Issue Number:**

Fixes #1156 

**Did you add tests for your changes?**

No

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

Screenshot (99)
Screenshot (99)

**Summary**

This PR fixes the issue #1156 

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by Min2who** [comment]
@aashimawadhwa @rishav-jha-mech can you review the PR

**Review by palisadoes — DISMISSED** [review]
Please add tests for the file you have edited. That is why the PR tests are failing.

The information can be found here
- https://github.com/PalisadoesFoundation/talawa-admin/pull/1170/files

**Screenshot**
image

**Comment by Min2who** [comment]
> Please add tests for the file you have edited. That is why the PR tests are failing.
> 
> The information can be found here
> 
> * https://github.com/PalisadoesFoundation/talawa-admin/pull/1170/files
> 
> **Screenshot** image

@palisadoes @aarishshahmohsin There is already a issue covering the test of this component issue #1167 
Should I make the test or wait for the above mentioned issue to be solved

**Comment by palisadoes** [comment]
1. Yes, edit the required files for this PR to get the code coverage up so that the tests pass and are valid tests.
2. We cannot allow the code base to revert to being under 95% coverage.

**Comment by Min2who** [comment]
@palisadoes @aashimawadhwa @EshaanAgg have added the test for it 
Screenshot 2023-12-11 105642

**Review by EshaanAgg — DISMISSED** [review]

**Comment by aashimawadhwa** [comment]
The advertisement delete toast should be of red color and did you changed the color of the button? @Min2who

**Comment by Min2who** [comment]
> The advertisement delete toast should be of red color and did you changed the color of the button? @Min2who

No i have not changed the color of the button

**Comment by aashimawadhwa** [comment]
Yes please change the color of button and add a confirm delete modal follow the pattern that is in delete post functionality on the posts section @Min2who

**Comment by Min2who** [comment]
> Yes please change the color of button and add a confirm delete modal follow the pattern that is in delete post functionality on the posts section @Min2who

@aashimawadhwa committed the changes that you have mentioned and updated the test but wonder why the type check is failing in the PR

**Comment by palisadoes** [comment]
We had to revert a PR that was causing tests to fail incorrectly. Please submit a minor commit to trigger an update of the GitHub workflow

File: public/locales/en.json [hunk 1]
```diff
@@ -679,7 +679,11 @@
     "EXname": "Ex. Cookie Shop",
     "EXlink": "Ex. http://yourwebsite.com/photo",
     "register": "Create Advertisement",
-    "close": "Close "
+    "close": "Close ",
+    "deleteAdvertisement": "Delete Advertisement",
+    "deleteAdvertisementMsg": "Do you want to remove this advertisement?",
+    "no": "No",
+    "yes": "Yes"
   },
```
[the remark this answers is not in the document]
Comment (Min2who) [reply public/locales/en.json:687]: is it not in all languages?

[the remark this answers is not in the document]
Comment (noman2002) [reply public/locales/en.json:687]: Please check other language files too.

[the remark this answers is not in the document]
Comment (noman2002) [reply public/locales/en.json:687]: As you have added new texts, you have add these in all other languages also.

**Comment by palisadoes** [comment]
@aashimawadhwa @EshaanAgg Please review

**Review by noman2002 — APPROVED** [review]
Looks fine to me.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**